universum/modules/output/html_output.py

```diff
@@ -15,68 +15,57 @@ def __init__(self, *args, **kwargs):
         self._filename = None
         self.artifact_dir_ready = False
         self._log_buffer = list()
+        self._block_level = 0
```

Comment:
Are we sure this does not duplicate block leveling, implemented in `structure_handler.py`?

Reply:
I'm not sure, I did this similar to `TerminalBasedOutput`. I will check later and reply.